tools/DevDiagnostics/DevHome.DevDiagnostics/Pages/InsightsPage.xaml

```diff
@@ -32,29 +32,26 @@
         <ItemsControl x:Name="InsightsItemsControl" ItemsSource="{x:Bind ViewModel.InsightsService.InsightsList}" Grid.Row="2">
             <ItemsControl.ItemTemplate>
                 <DataTemplate x:DataType="models:Insight">
-                    <Expander ExpandDirection="Down" HorizontalAlignment="Stretch" VerticalAlignment="Top" Padding="0,6" AutomationProperties.AutomationId="Expander">
+                    <Expander ExpandDirection="Down" HorizontalAlignment="Stretch" HorizontalContentAlignment="Left" VerticalAlignment="Top" Padding="0,6" AutomationProperties.Name="{x:Bind Title}">
                         <interactivity:Interaction.Behaviors>
                             <helpers:ExpanderBehavior />
                         </interactivity:Interaction.Behaviors>
-                        <Expander.Header>
-                            <Grid AutomationProperties.AutomationId="ExpanderHeader">
-                                <TextBlock Text="{x:Bind Title}" AutomationProperties.AutomationId="TitleTextBlock"/>
+                        <Expander.Header >
+                            <Grid>
+                                <TextBox Text="{x:Bind Title}" AutomationProperties.AutomationId="TitleTextBlock" IsReadOnly="True" BorderThickness="0" Background="Transparent"/>
```

Comment:
More auto-binding (I don't know the real name sorry) opportunity to simplify this?

Reply:
Not sure about auto-binding, but yes, I can create a "ReadOnlyTextBox" style and use that... so I did.